### DISCUSSION TITLE
Distributed PV dGEN Data

### DISCUSSION THREAD

**Discussion by tjfarrellag** [NREL/ReEDS-2.0#121] [Q&A: Data]
I’m reaching out to better understand the distributed roof top solar assumption for the ReEDS model. We are specifically interested in a breakdown of residential, commercial, and industrial distributed solar assumptions at the state-level for the Mid-Case Scenario, found at the link below. It is our understanding that the raw output of the dGen model that was used for the Mid-Case Scenario would have this breakdown. We were wondering if you could please help us find this data.  

Link to mid-case scenario: https://maps.nrel.gov/slope/data-viewer?layer=standard-scenarios.mid-re-cost&res=state&year=2020&filters=%5B%5D

**Comment by wesleyjcole** [comment]
The distributed PV assumptions are in inputs/dGen_Model_Inputs.  By the time they get to ReEDS, they have been rolled up across the three sectors into total distributed PV capacity by ReEDS model balancing area.
